### PR TITLE
Cindy dev

### PR DESCRIPTION
**Pull request by cindyke124**
## Description
Describe what changes you are making and why they are required
Added view_request page, users can filter their all requests by selecting request status and other conditions.

## How did you test the changes?
Describe how you tested the changes

## Testing Screenshots (Optional)
Add relevant screenshots for reference
Screenshot 2024-03-07 at 3 22 19 PM

### CODE REVIEW

**Comment by sarthakgoel1997** [comment]
@cindyke124 Please take a pull of the code on develop branch once

File: healthScore/templates/view_requests.html [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+                        <td>{{ request.appointment_type }}</td>
+                        <td>{{ request.doctor_name }}</td>
+                        <td>{{ request.hospital_name }}</td>
+                        <td>{{ request.hospital_address }}</td>
+                        <td>{{ request.createdAt }}</td>
+                        <td>{{ request.updatedAt }}</td>
+                        <td class="{% if request.record_status == 'approved' %}text-success
+                            {% elif request.record_status == 'rejected' %}text-danger
+                            {% else %}text-dark{% endif %}">
+                            {{ request.record_status }}
```
Comment (shreyas2499) [inline healthScore/templates/view_requests.html]: Could you please capitalize the first letter here?

**Review by shreyas2499 — APPROVED** [review]

File: healthScore/user_utils.py [hunk 2]
```diff
@@ -48,6 +48,11 @@ def get_health_history_details(request):
             ).values_list("id", flat=True)
             history_list = history_list.filter(hospitalID__in=hospital_ids)
 
+        # Filter records by status
+        record_status = request.GET.get("record_status")
+        if record_status:
+            history_list = history_list.filter(status=record_status)
+
```
Comment (sarthakgoel1997) [inline healthScore/user_utils.py]: I think these changes would be removed after taking a pull of the develop branch

File: healthScore/views.py [hunk 3]
```diff
@@ -47,6 +47,13 @@ def test_default_values(request):
 
 
 def view_health_history(request):
+    # Create a new QueryDict object with the desired parameters: fetch only approved records for health history page
+    updated_params = request.GET.copy()
+    updated_params["record_status"] = "approved"
+
+    # Update request.GET with the modified QueryDict
+    request.GET = updated_params
+
```
Comment (sarthakgoel1997) [inline healthScore/views.py]: These as well

**Review by sarthakgoel1997 — CHANGES_REQUESTED** [review]

File: healthScore/templates/view_requests.html (hunk 1 above)
Comment (cindyke124) [reply healthScore/templates/view_requests.html]: fixed

**Review by sarthakgoel1997 — APPROVED** [review]

**Pull request merged**